javascript/operators/precedence.json

```diff
@@ -0,0 +1,61 @@
+{
+  "javascript": {
+    "operators": {
+      "precedence": {
```

Comment:
Is this really a feature? Not convinced.

Reply:
I agree. It doesnt make sense. I'll remove this.